core/arch/arm/kernel/kern.ld.S

```diff
@@ -50,13 +50,19 @@
 
 #include <platform_config.h>
 
+#ifndef SMALL_PAGE_SIZE
+#define SMALL_PAGE_SIZE		4096
```

Comment:
Since this is introduced, please update all the other cases here where we align on small page boundary.